src/schema/v1/ordered_sets.ts

```diff
@@ -30,7 +30,10 @@ const OrderedSets: GraphQLFieldConfig<void, ResolverContext> = {
       defaultValue: 10,
     },
   },
-  resolve: (_root, options, { setsLoader }) => setsLoader(options),
+  resolve: async (_root, args, { setsLoader }) => {
+    const { body } = await setsLoader(args)
```

Comment:
So: why isn't it the case that we don't have a uniform return for all the loaders and just bulk change everything into using that? This seems potentially very confusing.

Reply:
Technical debt! I don't have a great answer except that at some point we wanted the headers too, and so needed to change it to `body` and `headers`, which wound up being opt-in (so many loaders _dont_ have that signature).

Meanwhile, the only reason we want the headers is b/c of our homegrown way of passing around counts like that! Which is b/c the list return type of a REST endpoint doesn't offer any additional metadata fields (like counts).

And so forth...

It's technical debt all the way down!

Reply:
Would there be any opposition to just bulk converting the entire codebase into the headers style return?

Reply:
I don't think so! More just...extra work, slog, tests to update, possible bugs, etc. (aka why technical debt persists).